src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -726,6 +728,7 @@ gasInterpreter :: Gas -> TransactionM db (Interpreter p)
 gasInterpreter g = do
     mc <- use txCache
     return $ initStateInterpreter
+        $ set evalLogGas (Just [("GTxSize",g)]) -- enables gas logging
```

Comment:
Is this fine to merge or does it create a lot of perf overhead?

Reply:
This doesn't. Although @LindaOrtega why is it `set` and not `over` with `(++)` or something? Are you sure `evalGasLog` is empty here?